.bazelrc

```diff
@@ -1,5 +1,4 @@
 common --enable_platform_specific_config
-common --enable_bzlmod
 # because we use --override_module with `%workspace%`, the lock file is not stable
 common --lockfile_mode=off
```

Comment:
not really related, but I remember discussing turning the lock file on and adding it to `.gitignore` to at least locally have the advantages of the lock file...

Reply:
In the internal repo that's what we're doing - let's do that separately from this PR.